src/components/VIconButton/VIconButton.vue

```diff
@@ -5,7 +5,11 @@
     :type="type"
     v-on="$listeners"
   >
-    <VIcon :class="[...iconSizeClasses]" v-bind="iconProps" />
+    <VIcon
+      class="pointer-events-none"
```

Comment:
This allows icon click events to pass through to their parent element, which makes sure the `target` of icon buttons clicks is always the button, not the internal icon.